locales/en/messages.json

```diff
@@ -2755,11 +2755,11 @@
     "dataflashButtonSaveFile": {
         "message": "Save flash to file..."
     },
-    "dataflashButtonSaveFileDeprecated": {
-        "message": "Save flash to file... (deprecated)"
+    "dataflashSavetoFileNote": {
+        "message": "Directly saving flash to file is convenient for small files, but is slow and inherently prone to error / file corruption.<br>If this method fails, do not log a support request - use Mass Storage mode instead."
     },
     "dataflashSaveFileDepreciationHint": {
-        "message": "Saving blackbox logs through the configurator is slow and relatively prone to errors. It is recommended that you instead use '<b>$t(onboardLoggingRebootMscText.message)</b>' (below) to activate the Mass Storage Mode, and access your flight controller as a storage device to download the log files."
+        "message": "This method is slow and inherently prone to error / file corruption. It's suitable for small log files only. Do not create support requests if file transfers fail when saved using this method. The recommended method is to use '<b>$t(onboardLoggingRebootMscText.message)</b>' (below) to activate the Mass Storage Mode, and access your flight controller as a storage device to download the log files."
```

Comment:
Replace 'It is suitable' with 'It may work' - this design is not suitable for any use.